src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -546,13 +593,14 @@ bool Dataloader::_load_map_dir(GameManager& game_manager, fs::path const& map_di
 		),
 
 #define MAP_PATH_DICT_ENTRY(X) \
-		#X, ONE_EXACTLY, expect_string(assign_variable_callback(X)),
+	#X, ONE_EXACTLY, expect_string(assign_variable_callback(X)),
 		APPLY_TO_MAP_PATHS(MAP_PATH_DICT_ENTRY)
 #undef MAP_PATH_DICT_ENTRY
 
 #undef APPLY_TO_MAP_PATHS
 
-		"border_heights", ZERO_OR_ONE, success_callback,
+			"border_heights",
+		ZERO_OR_ONE, success_callback,
```

Comment:
Why is this being moved to two lines by the formatter?

Reply:
Unsure, fixed it in the file. Not too familiar with clang-format.